src/main/java/org/pac4j/dropwizard/Pac4jFactory.java

```diff
@@ -135,6 +138,28 @@ public void setCallbackUrlResolver(
         this.callbackUrlResolver = callbackUrlResolver;
     }
 
+    /**
+     * @since 1.1.0
+     */
+    @JsonProperty
+    public Boolean getSessionEnable() {
+        return sessionEnable;
+    }
+
+    /**
+     * 
+     * @param sessionEnable
+     *            if <code>true</code> session will be enabled at Jetty level
+     *            not matter what, if <code>false</code> it won't no matter what
+     *            and if <code>null</code> then sessions will be enabled only if
+     *            there is {@link IndirectClient} in {@link #getClients()}.
```

Comment:
Is there a better way to model this than using a boolean as a nullable tri-state?

Taking a step back, do we have to support the implicit "enabled if an indirect client exists" behavior? It seems to me that using a simple enabled/disabled flag would a) simplify the configuration options and b) prevent any confusion caused by the implicit dependency on the presence of indirect clients.